lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/Bitdefender.pm

```diff
@@ -0,0 +1,81 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::Bitdefender;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use UNIVERSAL::require;
+use Cpanel::JSON::XS;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return canRun('bduitool');
+}
+
+sub doInventory {
+    my (%params) = @_;
+
+    my $inventory = $params{inventory};
+    my $logger    = $params{logger};
+
+    my $antivirus = _getBitdefenderInfo(logger => $logger);
+    if ($antivirus) {
+        $inventory->addEntry(
+            section => 'ANTIVIRUS',
+            entry   => $antivirus
+        );
+
+        $logger->debug2("Added $antivirus->{name}" . ($antivirus->{version} ? " v$antivirus->{version}" : ""))
+            if $logger;
+    }
+}
+
+sub _getBitdefenderInfo {
+    my (%params) =  (
+        command => '/opt/bitdefender-security-tools/bin/bduitool get ps',
```

Comment:
If the command is in the PATH, you don't need to use a full path. Indeed, you must be coherent with the `canRun()` call made in `isEnabled()` method.

Reply:
A full path is needed.